### PR TITLE
FWF-4078: Added input validate loader for form title

### PR DESCRIPTION
**Pull request by Bonymol-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-4078
Issue Type: BUG/ FEATURE

# Changes
* Added input validate loader for form title in settings

# Screenshots (if applicable)

# Notes

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

File: forms-flow-web/src/components/Form/EditForm/FormSettings.js [hunk 1]
```diff
@@ -211,6 +219,7 @@ const FormSettings = forwardRef((props, ref) => {
           ariaLabel={t("Form Name")}
           isInvalid = {!!errors.name}
           feedback = {errors.name}
+          isValidating={isValidating.name}
```
Comment (arun-s-aot) [inline forms-flow-web/src/components/Form/EditForm/FormSettings.js:222]: change prop to turnOnLoader

Comment (Bonymol-aot) [reply forms-flow-web/src/components/Form/EditForm/FormSettings.js]: Resolved

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Form/EditForm/FormEdit.js [hunk 2]
```diff
@@ -1156,7 +1156,8 @@ const EditComponent = () => {
         nameLabel={t("New Form Name")}
         descriptionLabel={t("New Form Description")}
         showBuildForm={selectedAction === DUPLICATE}
-        isLoading={formSubmitted || validationLoading}
+        isSaveBtnLoading={formSubmitted}
+        isFormNameValidating={validationLoading}
```
Comment (arun-s-aot) [inline forms-flow-web/src/components/Form/EditForm/FormEdit.js:1160]: we will need similar change in List.js page also right ? @Bonymol-aot

Comment (Bonymol-aot) [reply forms-flow-web/src/components/Form/EditForm/FormEdit.js:1160]: Missed it while pushing, updated 👍

**Review by arun-s-aot — APPROVED** [review]

**Pull request merged**